client/core/trade.go

```diff
@@ -442,20 +435,7 @@ func (t *trackedTrade) isRedeemable(match *matchTracker) bool {
 		return false
 	}
 
-	wallet := t.wallets.toWallet
-	if !wallet.unlocked() {
-		log.Errorf("cannot redeem order %s, match %s, because %s wallet is not unlocked",
-			t.ID(), match.id, unbip(wallet.AssetID))
-		return false
-	}
-
-	walletLocked := !t.wallets.toWallet.unlocked()
```

Comment:
I didn't mean to get rid of the first check. I just meant that there's no need to keep checking after that. The `walletLocked` variable would always be `false`, since you just verified it a few lines up.

Reply:
Oh. Thought you meant all checks. The checks are not too relevant anyways since `core` tries to keep the wallets unlocked while there are active trades. But a user may manually lock the wallet, so restoring the checks just to be safe.